common/views/components/EventPromo/README.md

```diff
@@ -0,0 +1,7 @@
+Event Promo
+
+image 16:9
+
+what is displayed
+what is always displayed
+what is optionally displayed
```

Comment:
It would be way nicer to use types for this, and not leave us with loads to update or forget about.

Reply:
Agree, was just putting the READMEs in as placeholders for this PR

Reply:
From my previous mistakes, I'm trying not to put placeholders in as they tend to live longer than you might expect.

Reply:
I've started going with `__TODO:__ put README in this file` – thoughts?

Reply:
I know there is a default README missing message.
Maybe we can find a way to update that?